python_modules/dagster/dagster/_core/execution/context/input.py

```diff
@@ -513,16 +537,28 @@ def test_load_input():
         """
         return self._observations
 
-    def consume_metadata(self) -> Mapping[str, MetadataValue]:
-        result = self._metadata
-        self._metadata = {}
+    def consume_logged_metadata(self) -> Mapping[str, MetadataValue]:
```

Comment:
renamed this method to align with `OutputContext`. Can revert if we are worried users are calling this method. But it's not part of the public API so i think it should be fine to rename

Reply:
is there a reason for users to call this ? seems like its mostly for internal use so i think its fine to change

Reply:
yeah it's just internal and i dont know of a reason it would be called